src/swift/Data.swift

```diff
@@ -156,19 +193,41 @@ public struct DispatchData : RandomAccessCollection {
 	/// - parameter pointer: A pointer to the buffer you wish to copy the bytes into.
 	/// - parameter count: The number of bytes to copy.
 	/// - warning: This method does not verify that the contents at pointer have enough space to hold `count` bytes.
+	@available(swift, deprecated: 4, message: "Use copyBytes(to: UnsafeMutableRawBufferPointer, count: Int) instead")
 	public func copyBytes(to pointer: UnsafeMutablePointer<UInt8>, count: Int) {
 		_copyBytesHelper(to: pointer, from: 0..<count)
 	}
+
+	/// Copy the contents of the data to a pointer.
+	///
+	/// - parameter pointer: A pointer to the buffer you wish to copy the bytes into.
+	/// - parameter count: The number of bytes to copy.
+	/// - warning: This method does not verify that the contents at pointer have enough space to hold `count` bytes.
```

Comment:
How do you know if it copied the whole thing? I'd prefer to assert, so that clients are encouraged to check the length beforehand.